<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RR</class>
 <widget class="QWidget" name="RR">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>517</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="4">
      <widget class="QSpinBox" name="stopTimeSpinBox"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>p</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" rowspan="4" colspan="5">
      <widget class="QWidget" name="mplWidget" native="true">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>100</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Nmax </string>
       </property>
      </widget>
     </item>
     <item row="7" column="4">
      <widget class="QPushButton" name="stopPushButton">
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
     </item>
     <item row="7" column="3">
      <widget class="QPushButton" name="pausePushButton">
       <property name="text">
        <string>Pause</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0" colspan="2">
      <widget class="QPushButton" name="startPushButton">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3" colspan="2">
      <widget class="QPushButton" name="generatePushButton">
       <property name="text">
        <string>Generate</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="pDoubleSpinBox"/>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Stop time (min)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="nMaxSpinBox"/>
     </item>
     <item row="0" column="5" rowspan="8" alignment="Qt::AlignHCenter">
      <widget class="QListView" name="listView"/>
     </item>
     <item row="0" column="2" rowspan="2">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>RR</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="riLabel">
       <property name="text">
        <string>60</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Stop trial</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QSpinBox" name="stopTrialSpinBox"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <includes>
  <include location="local">import PyQt5</include>
 </includes>
 <resources/>
 <connections/>
</ui>
